### PR TITLE
fix(deps): removed doctrine/annotations and use

### PR DESCRIPTION
**Pull request by lu1a**
We have at least one customer who has problems with doctrine/annotations. Why did we even have it required anyway...

### CODE REVIEW

**Review by thevilledev — APPROVED** [review]
LGTM 👍

File: composer.json [hunk 1]
```diff
@@ -18,21 +23,23 @@
...
     "symfony/serializer": "^5.1",
     "symfony/property-access": "^5.1",
-    "webmozart/assert": "^1.9",
-    "doctrine/annotations": "^1.11"
+    "webmozart/assert": "^1.9"
   },
   "require-dev": {
     "phpunit/phpunit": "^8.5.8",
     "squizlabs/php_codesniffer": "~3.5",
-    "friendsofphp/php-cs-fixer": "~2.18.3",
```
Comment (Darep) [inline composer.json:27]: Was this removed because it has a require for annotations as well?

Comment (lu1a) [reply composer.json:27]: yessir seemingly

**Review by Darep — APPROVED** [review]

**Comment by lu1a** [comment]
Okie dokie someone with write access can merge whenever. 😁

**Pull request merged**